src/components/CurrencyInputPanel/FiatValue.tsx

```diff
@@ -35,11 +36,19 @@ export function FiatValue({
     return <FiatLoadingBubble />
   }
 
+  let formattedValue = formatNumber(fiatValue.data, NumberType.FiatTokenPrice)
+  if (formattedValue.length > 18) {
+    //Removes the dollar sign at the start, commas, and decimal places
+    formattedValue = formattedValue.slice(1).replaceAll(',', '').slice(0, -4)
```

Comment:
several countries use `,` instead of a decimal point

Reply:
Would localization issues be bypassed if I just updated this in `conedison` like @tinaszheng mentioned?

Reply:
yeah would be easier to let the `conedison` formatter take care of it since it allows us to pass in a `currency` type into the formatter